tests/test_cli.py

```diff
@@ -625,6 +625,18 @@ def test_list_outputs_with_export(self):
         assert result.exit_code == 0
         assert result.output == "export SCEPTRE_Key='Value'\n"
 
+    @pytest.mark.parametrize("output_format,expected_output", [
+        ("yaml", '---\n- mock-stack.yaml: mock-stack-external\n\n'),
+        ("text", 'mock-stack.yaml: mock-stack-external\n'),
+        ("json", '[\n    {\n        "mock-stack.yaml": "mock-stack-external"\n    }\n]\n')
```

Comment:
Yeah, it would look a lot better if you just made a single dict rather than a list of single-key dicts.